spec/acceptance/suites/rpm_docker/nodesets/default.yml

```diff
@@ -44,11 +55,15 @@ HOSTS:
       - 'yum install -y epel-release'
       - 'yum install -y openssl util-linux rpm-build augeas-devel createrepo genisoimage git gnupg2 libicu-devel libxml2 libxml2-devel libxslt libxslt-devel rpmdevtools clamav clamav-update which ruby-devel rpm-devel rpm-sign'
 
+      # fakesystemd is removed and base dependencies are installed,
+      # so it's possible to 'upgrade' openssl
+      - 'yum localinstall -y openssl*.rpm'
```

Comment:
This should not be required any longer with the changes that @op-ct added to build an internal `rvm` openssl.